skimage/_shared/utils.py

```diff
@@ -208,7 +208,7 @@ def __init__(self, kwarg_mapping, deprecated_version, warning_msg=None,
                                 "for `{func_name}`. ")
             if removed_version is not None:
                 self.warning_msg += (f'It will be removed in '
-                                     f'version {removed_version}.')
+                                     f'version {removed_version}. ')
```

Comment:
@rfezzani please cherry-pick this for #6183 :wink:

Reply:
Done, thank you :wink: